web/src/main/kotlin/no/nav/su/se/bakover/web/routes/person/PersonRoutes.kt

```diff
@@ -65,8 +73,52 @@ internal fun Route.personRoutes(
             )
         }
     }
+
+    get("$personPath/{fnr}/sakinfo") {
+        call.parameter("fnr")
+            .flatMap {
+                Either.catch { Fnr(it) }
+                    .mapLeft { Feilresponser.ugyldigFødselsnummer }
+            }
+            .map { fnr ->
+                sakService.hentSak(fnr)
```

Comment:
Synes vi bør lage en ny funksjon i sakService som gjør denne logikken og at *Routes.kt kan stå for det som har med http og konvertering av json til primitiver/objekter/lister (holde den relativt dum).